### PR TITLE
C#: Adjust make_stubs.py to use codeql instead of odasa

### PR DESCRIPTION
**Pull request by tamasvajk**

### CODE REVIEW

**Review by hvitved — DISMISSED** [review]
Looks plausible. Does it work? :-)

**Comment by tamasvajk** [comment]
> Looks plausible. Does it work? :-)

Good question. I did minimal testing. It generates a stub file with meaningful content. 

I tried it on a couple of more test folders now, and it looks like I need to improve the file trimming a bit.

File: csharp/ql/src/Stubs/make_stubs.py [hunk 1]
```diff
@@ -58,52 +52,71 @@
...
+    print('It may help to view', outputFile, ' in Visual Studio')
+    print("Next steps:")
+    print('1. Look at the compilation errors, and fix up',
+          outputFile, 'so that the test compiles')
+    print('2. Re-run codeql test run "' + testDir + '"')
+    print('3. git add "' + outputFile + '"')
+    exit(1)
 
 print("\nStub generation successful! Next steps:")
 print('1. Edit "semmle-extractor-options" in the .cs files to remove unused references')
```
Comment (tamasvajk) [inline csharp/ql/src/Stubs/make_stubs.py:122]: There's still some reference to `odasa` one line below.

**Comment by tamasvajk** [comment]
@hvitved I successfully used this to generate stubs for Dapper.

**Review by hvitved — APPROVED** [review]

**Pull request merged**